src/Utils/Helpers.ts

```diff
@@ -67,27 +67,24 @@ export class RealPath {
 
 /**
  * @brief Get Workspace root folder as string
- * @note  will throw if not working in workspace mode.
+ * @return Return only the first workspaceFolder if multiple root exists, with showing a notfication
+ *     balloon.
+ *         Throws if there is no workspace.
  */
 export function obtainWorkspaceRoot(): string {
   const workspaceFolders = vscode.workspace.workspaceFolders;
-  if (!workspaceFolders) {
-    Logger.error(logTag, 'obtainWorkspaceRoot: NO workspaceFolders');
-    // TODO revise message
```

Comment:
`Need workspace` is as-is but removed this `TODO`. Do we not need to change this message?
IMHO, this looks not friendly with short `Need workspace`.

Reply:
There are so many tests that checks this 'Need workspace' message. And I thought that as many people remained it, it's good to keep as is. However if you allow, we can revise it to `There is no workspace.`, which looks rather formal.

Reply:
> And I thought that as many developers kept it while using it

Seems this is another context. It would be better to remove this `TODO` in another PR.

> However if you allow

I don't think I'm the person to allow or not for this message.

> we can revise it to There is no workspace. in the next PR, which looks rather formal.

This seems a choice to revise the mssage.

Anyway, for this PR, it would be better not to remove `TODO` message in this PR.

Reply:
If you don't mean to revise the message, then it's no meaning that the comment remain there. Anyways I will revive the TODO message then.